cfn-publish.config

```diff
@@ -1,3 +1,3 @@
-template=cfn.template
+templates=cfn.template
```

Comment:
I would guess this could break cfn-publish if it's being used to publish itself. Is it?

Reply:
it is

Reply:
Ok, I'll roll this back and can re-add it later - after it's deployed.

Reply:
Done